### DISCUSSION TITLE
gPRC transcoding for REST API

### DISCUSSION THREAD

**Discussion by emcfarlane** [loxilb-io/loxilb#272] [Ideas]
👋 This is an idea to switch the REST API to use gRPC transcoding annotations. I've been working on a project to help transcode called [larking](https://github.com/emcfarlane/larking). Have you looked into it before? What benefits do you get from the current swagger style APIs?

**Comment by inhogog2** [comment]
Hi. You're working on a great project! Unfortunately, we don't have any plans to support gRPC at the moment, but I'll refer to it if we support gRPC in the future. Thank you for letting me know. The reason why we use Swagger is that we must write a document before we start coding. So we are sticking to the Swagger style.

**Reply by emcfarlane** [reply]
Thanks for the insight! You're project is great and I wanted to help contribute so If you ever look at switching API schemes happy to help.
